utils/roll_browser.js

```diff
@@ -100,6 +100,10 @@ Example:
       const headlessShellBrowser = await browsersJSON.browsers.find(b => b.name === 'chromium-headless-shell');
       headlessShellBrowser.revision = revision;
       headlessShellBrowser.browserVersion = browserVersion;
+    } else if (browserName === 'chromium-tip-of-tree') {
+      const tipOfTreeBrowser = await browsersJSON.browsers.find(b => b.name === 'chromium-tip-of-tree-headless-shell');
```

Comment:
Why can't we just user version of `chromium-tip-of-tree` ? We don't expect these two to ever diverge, do we?

Reply:
I'll follow-up.